PAT.py

```diff
@@ -3,15 +3,16 @@
 from numpy.random import mtrand
 import pygame
 from configReader import ConfigReader, ConfigContainer
-from src.io import LogWriter
+from src.pat_io import LogWriter
```

Comment:
just curious, what sort of warning were you getting with `io`? (I suspect the "right" way to do this is to set up a proper `setup.py` file and do a `pip install -e .` of the package instead of relying explicitly on local directory structure.)

Reply:
I was getting a warning from vscode about overriding stdlib io

Reply:
Oh interesting. I have no idea whether it's better to avoid namespace conflicts like that or just ignore the warning https://stackoverflow.com/a/74675579/2232265. 

FWIW, gpt agrees with you :)  
![image](https://user-images.githubusercontent.com/4934167/206007290-0f5c410a-cf9a-4925-92d1-0ffd8c1c72a7.png)